### DISCUSSION TITLE
Set default browser to a specific application

### DISCUSSION THREAD

**Discussion by adrianoresende** [will-stone/browserosaurus#504] [Q&A] [answered]
Hi there,
Thank you for this feature that "improves life" for those who have two browsers to separate work and personal activities.

I use Slack to work, but is it possible to set the browser to a specific application without opening the popup to choose every I click the link in Slack?

**Comment by will-stone** [comment, marked as answer]
Hi @adrianoresende Thank you for the kind words. Please see the answer to _["I would like to open a particular browser automatically for specific URLs"](https://github.com/will-stone/browserosaurus/discussions/476)_, as the answer is the same 🙂

**Reply by will-stone** [reply]
In fact, I'll update the title of that discussion to include "from specific apps". Thanks
